### PR TITLE
mysql-schema: add description for MDL related table

### PR DESCRIPTION
**Pull request by wjhuang2016**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: mysql-schema.md [hunk 1]
```diff
@@ -66,6 +66,10 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 * `tidb_runaway_watch`：Runaway Queries 的监控列表 (Watch List)
 * `tidb_runaway_watch_done`：被删除或者过期的 Runaway Queries 的监控列表
 
+## 元数据锁相关系统表
+* `tidb_mdl_view`：元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
+* `tidb_mdl_info`：内部用于同步各节点的元数据锁
```
Comment (hfxsd) [inline mysql-schema.md:71]:
```suggestion
## 元数据锁相关系统表

* `tidb_mdl_view`：元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
* `tidb_mdl_info`：内部用于同步各节点的元数据锁
```

**Review by hfxsd — APPROVED** [review]

**Review by okJiang — APPROVED** [review]

File: mysql-schema.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 * `tidb_runaway_watch`：Runaway Queries 的监控列表 (Watch List)
 * `tidb_runaway_watch_done`：被删除或者过期的 Runaway Queries 的监控列表
 
+## 元数据锁相关系统表
+
+* `tidb_mdl_view`：元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
+* `tidb_mdl_info`：内部用于同步各节点的元数据锁
```
Comment (qiancai) [inline mysql-schema.md:72]:
```suggestion
* `tidb_mdl_view`：元数据锁的视图，可以用于查看当前阻塞的 DDL 的相关信息
* `tidb_mdl_info`：TiDB 内部用于同步各节点的元数据锁的相关信息
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**